src/Infrastructure/IFileFacade.cs

```diff
@@ -217,7 +218,7 @@ public interface IFileFacade
         /// <exception cref="T:System.Security.SecurityException">
         ///   The caller does not have the required permission.
         /// </exception>
-        void WriteAllText(string path, string contents);
```

Comment:
We can’t remove the old method. That would break all clients. Instead, this should be a new method of the interface and keep the old one.

Reply:
Added a new overload for WriteAllText keeping the existing one.